lib/style_checker.rb

```diff
@@ -0,0 +1,17 @@
+require 'octokit'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.